.changelog/41379.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/aws_paymentcryptography_key: Set `key_attributes` and `key_attributes.*.key_modes_of_use` arguments to required
```

Comment:
Confirmed these are required by the AWS API.

- https://docs.aws.amazon.com/payment-cryptography/latest/APIReference/API_CreateKey.html#paymentcryptography-CreateKey-request-KeyAttributes
- https://docs.aws.amazon.com/payment-cryptography/latest/APIReference/API_KeyAttributes.html#paymentcryptography-Type-KeyAttributes-KeyModesOfUse